docs/llm/reference.md

```diff
@@ -0,0 +1,147 @@
+
+
+The value of `cache_size` might have performance  implications. It is used for storing LLM model KV cache data. Adjust it based on your environment capabilities, model size and expected level of concurrency.
+
+## Models Directory
+
+In node configuration we set `models_path` indicating location of the directory with files loaded by LLM engine. It loads following files:
+
+```
+├── openvino_detokenizer.bin
+├── openvino_detokenizer.xml
+├── openvino_model.bin
+├── openvino_model.xml
+├── openvino_tokenizer.bin
+├── openvino_tokenizer.xml
+├── tokenizer_config.json
+├── template.jinja
+```
+
+Main model as well as tokenizer and detokenizer are loaded from `.xml` and `.bin` files and all of them are required. `tokenizer_config.json` and `template.jinja` are loaded to read information required for chat template processing. Chat template is used only on `/chat/completions` endpoint. Template is not applied for calls to `/completions`, so it doesn't have to exist, if you plan to work only with `/completions`. 
+
+### Chat template
+
+Loading chat template proceeds as follows:
+1. If `tokenizer.jinja` is present, try to load template from it.
+2. If there is no `tokenizer.jinja` and `tokenizer_config.json` exists, try to read template from its `chat_template` field. If it's not present, use default template.
+3. If `tokenizer_config.json` exists try to read `eos_token` and `bos_token` fields. If they are not present, both values are set to empty string. 
+
+**Note**: If both `template.jinja` file and `chat_completion` field from `tokenizer_config.json` are successfully loaded `template.jinja` takes precedence over `tokenizer_config.json`.
+
+If there are errors in loading or reading files or fields (they exist but are wrong) no template is loaded and servable will not respond to `/chat/completions` calls. 
+
+If no chat template has been specified, default template is applied. The template looks as follows:
+```
+"{% if messages|length > 1 %} {{ raise_exception('This servable accepts only single message requests') }}{% endif %}{{ messages[0]['content'] }}"
+```
+
+When default template is loaded, servable accepts `/chat/completions` calls when `messages` list contains only single element (otherwise returns error) and treats `content` value of that single message as an input prompt for the model.
+
+
+## Limitations
+
+As it's in preview, this feature has set of limitations:
+
+- Limited support for [API parameters](./model_server_rest_api_chat.md#request),
+- Only completions and chat completions endpoints supported,
+- Only one node with LLM calculator can be deployed at once,
+- Lack of metrics,
+- Not all performance optimizations yet applied,
+- Improvements in stability and recovery mechanisms are also expected,
```

Comment:
Maybe it would be a place to mention about tested models? 
As a preview feature it is made available for customers to evaluate and provide feedback.